tests/31sync/09archived.pl

```diff
@@ -219,7 +222,7 @@
 
       my $filter = {
          room => {
-            timeline => { types => [ "m.room.message" ] },
+            timeline => { types => [ "m.room.message", "a.madeup.test.state" ] },
```

Comment:
This might have been neater using `qw( ... )` but then so many of the other tests aren't using that, so... hmmm